src/util/macros.rs

```diff
@@ -105,10 +105,7 @@ macro_rules! box_err {
 macro_rules! slow_log {
     ($t:expr, $($arg:tt)*) => {{
         if $t.is_slow() {
-            warn!("handle task slow";
-                "task" => format!("{}", format_args!($($arg)*)),
-                "elapsed" => ?$t.elapsed(),
-            );
+            warn!($($arg)*; "takes" => ?$t.elapsed());
```

Comment:
What if we want log more `kv`s?

Reply:
So we can extend the macro when we need this.